src/couch/src/couch_server.erl

```diff
@@ -95,8 +95,6 @@ get_spidermonkey_version() -> list_to_binary(?COUCHDB_SPIDERMONKEY_VERSION).
 sup_start_link(N) ->
     gen_server:start_link({local, couch_server(N)}, couch_server, [N], []).
 
--spec open(any(), [any()]) ->
```

Comment:
don't remove the spec, please. Add auxiliary type instead. 

```
-type throw(_Reason) :: no_return().

....

-spec open(any(), [any()]) -> 
     throw({error, all_dbs_active}).
```

Reply:
Bad delete. Added back with the auxiliary type.